src/modules/navigator/mission.cpp

```diff
@@ -362,6 +406,7 @@ Mission::set_execution_mode(const uint8_t mode)
 					pos_sp_triplet->current.type = position_setpoint_s::SETPOINT_TYPE_POSITION;
 					publish_navigator_mission_item(); // for logging
 					_navigator->set_position_setpoint_triplet_updated();
+					cacheItem(_mission_item);
```

Comment:
@sfuhrer Can you make a comment why this is necessary here?

Reply:
Well I was actually wanting to ask you the exact same question :sweat_smile: . I actually don't see where it comes from. Reverse mission is only something that's supported in RTL, no? And there we don't care about mission resume camera logic..

Reply:
removed it.